src/hgraph/_wiring/_wiring_node_class/_stub_wiring_node_class.py

```diff
@@ -24,3 +24,10 @@ class NonPeeredWiringNodeClass(StubWiringNodeClass):
 
     def __call__(self, _tsb_meta_type: HgTSBTypeMetaData, **kwargs) -> "WiringPort":
         ...
+
```

Comment:
This was an idea to use a stub node to rank components using the context, but I don't think this will remain as it did not fit well and the phantom inputs (rank_marker) seems like a cleaner approach to the problem.